pkg/investigations/chgm/chgm_hibernation_check.go

```diff
@@ -0,0 +1,87 @@
+package chgm
+
+import (
+	"sort"
+	"time"
+
+	v1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	servicelogsv1 "github.com/openshift-online/ocm-sdk-go/servicelogs/v1"
+	"github.com/openshift/configuration-anomaly-detection/pkg/ocm"
+)
+
+const RECENT_WAKEUP_TIME = 2 * time.Hour
+
+// 30 Days is always a problem as kubelet certificates will be expired
+const HIBERNATION_TOO_LONG = 30 * 24 * time.Hour
+const HIBERNATION_START_EVENT = "cluster_state_hibernating"
+const HIBERNATION_END_EVENT = "cluster_state_ready"
+const HIBERNATION_ONGOING_EVENT = "cluster_state_hibernating"
+const HIBERNATION_RESUME_EVENT = "cluster_state_resuming"
+
+type HibernationPeriod struct {
```

Comment:
**Question:** why does this have public?

Reply:
I kind of forgot that all of those are in the same package, so I made all of them private now.